UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling/ChangelingAbility/Implementation/Sting.cs

```diff
@@ -16,7 +16,7 @@ public class Sting: ChangelingBaseAbility
 		[SerializeField] protected float stingTime = 4f;
 		public float StingTime => stingTime;
 
-		protected PlayerScript GetPlayerOnClick(ChangelingMain changeling, Vector3 clickPosition, Vector3 rounded)
+		protected PlayerScript GetPlayerOnClick(ChangelingMain changeling, Vector3 clickPosition, Vector3 rounded, string messageWhenTargetIsDead = "<color=red>You cannot sting a dead body!</color>")
```

Comment:
I'm not sure I like the text being here. This method is called ``GetPlayerOnClick``, adding the messages here adds more responsibility to the method than its initial intention.

Reply:
Ok, gonna change that